src/components/chart/_macro.spec.js

```diff
@@ -240,6 +241,61 @@ describe('Macro: Chart', () => {
                 });
             });
         });
+
+        describe('GIVEN: Params: Legend', () => {
```

Comment:
Here we only need one given for these 4 tests

Reply:
Sorted.